packages/builders/src/messages/attachment/UnsafeAttachment.ts

```diff
@@ -0,0 +1,186 @@
+import type { Stream } from 'node:stream';
+import type { APIAttachment, Snowflake } from 'discord-api-types/v10';
+
+export type BufferResolvable = typeof ArrayBuffer | string;
```

Comment:
1. Small code style

```suggestion
export type BufferResolvable = typeof ArrayBuffer | string;

```

---

2. Are you sure its `typeof ArrayBuffer` and not `ArrayBuffer`?